account_invoice_inter_company_queued/models/account_move.py

```diff
@@ -8,7 +8,9 @@ class AccountMove(models.Model):
     def _post(self, soft=True):
         self = self.with_context(account_invoice_inter_company_queued=True)
         res = super(AccountMove, self)._post(soft=soft)
-        dest_company = self._find_company_from_invoice_partner()
-        if dest_company and not self.auto_generated:
-            self.sudo().with_delay().create_counterpart_invoices()
+        supported_types = {"out_invoice", "in_invoice", "out_refund", "in_refund"}
+        for src_invoice in self.filtered(lambda x: x.move_type in supported_types):
```

Comment:
One thing: why not using `_is_invoice` method (or similar name) instead of this?

Reply:
i did not know this method. updated.